pkg/apis/openstackproviderconfig/v1alpha1/types.go

```diff
@@ -203,6 +203,23 @@ type OpenstackClusterProviderSpec struct {
 	// to get public internet to the VMs.
 	ExternalNetworkID string `json:"externalNetworkId,omitempty"`
 
+	// ManagedAPIServerLoadBalancer defines whether a LoadBalancer for the
+	// APIServer should be created. If set to true the following properties are
+	// mandatory: APIServerLoadBalancerFloatingIP, APIServerLoadBalancerPort
+	ManagedAPIServerLoadBalancer bool `json:"managedAPIServerLoadBalancer"`
+
+	// APIServerLoadBalancerFloatingIP is the floatingIP which will be associated
+	// to the APIServer loadbalancer. The floatingIP will be created if it not
+	// already exists.
+	APIServerLoadBalancerFloatingIP string `json:"apiServerLoadBalancerFloatingIP,omitempty"`
+
+	// APIServerLoadBalancerPort is the port on which the listener on the APIServer
+	// loadbalancer will be created
+	APIServerLoadBalancerPort int `json:"apiServerLoadBalancerPort,omitempty"`
```

Comment:
if this is port id, type should be string.

Reply:
That's not a port-id it's the actual port value